app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -22,9 +22,9 @@ def call
 
       private
 
-      def create_authentication_parameters
+      def authentication_parameters
         @logger.debug(LogMessages::Authentication::AuthnJwt::CREATING_AUTHENTICATION_PARAMETERS_OBJECT.new)
```

Comment:
this log message should really go in the `@jwt_configuration.authentication_parameters` method as now the log message will be written each time `authentication_parameters` is called although it is memoized and created only once

Reply:
Done